vigir_pluginlib/PluginDescription filter  # filtering options (optional)
---
vigir_pluginlib/PluginDescription[] plugin_descriptions
